docs/terminologies.md

```diff
@@ -0,0 +1,54 @@
+# Terminologies
+RareCamp operates in the domain of biotechnology / drug discovery that is unfamiliar to most software developers.
+This document aims to standardize the terminologies for use within the codebase as well as communication among the
+developers. If there is any ambiguity, this document serves as the final authority:
+
+### Disease
```

Comment:
dumb question: Are all conditions mappable to diseases? I'm wondering if 'condition' is a better terminology here..